sources/osg/FrameBufferObject.js

```diff
@@ -216,7 +218,14 @@ FrameBufferObject.prototype = MACROUTILS.objectInherit( GLObject.prototype, MACR
         var gl = this._gl;
 
         // apply on unit 1 to init it
-        state.applyTextureAttribute( 1, texture );
+        // make sure we do bind it whatever state stack
+        // texture is cached
+        state.applyTextureAttribute( 1, texture, true );
```

Comment:
I prefer to have code if it makes sense. The just in case could mask something else. Could you remove it plz